diagnostic reports/README.md

```diff
@@ -0,0 +1,374 @@
+| text [@mediaType]                              | fixed value = "text/x-h7uk-pmip"        |
+| text                                           | EDIFACT comment, as described below     |
+| statusCode                                     | fixed value = `"COMPLETE"`              |
+| availabilityTime [@value]                      | `DiagnosticReport.issued`               |
+
+#### Text Element Mapping
+
+ Narrative Statements are created in relation to each of the FHIR Fields / Logical Reasons below. The value of the field 
+ is mapped to the Narrative Statement's `text` element as EDIFACT comment (see example in the XML above), the comment type 
+ of which is shown in the table:
+
+| FHIR field / logical reason                                                                                                                                                                | EDIFACT Comment Type                                    |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|
+| `diagnosticReport.conclusion`                                                                                                                                                              | `LABORATORY RESULT COMMENT(E141)`                       |
+| concatenated from `dianosticReport.codedDiagnosis`                                                                                                                                         | `LABORATORY RESULT COMMENT(E141)`                       |
+| `dianosticReport.status`                                                                                                                                                                   | `LABORATORY RESULT COMMENT(E141)`                       |
+| none of the fields above are populated and `dianosticReport.result` is missing                                                                                                             | `AGGREGATE COMMENT SET` and fixed body = `EMPTY REPORT` |
+| `Observation.EffectiveDateTime` or `Observation.EffectivePeriod` where the Observation <br/> is a referenced in `dianosticReport.result` and the first Observation coded as `Comment note` | `AGGREGATE COMMENT SET`                                 |
+| Concatenated from `diagnosticReport.performer[index].name`                                                                                                                                 | `AGGREGATE COMMENT SET`                                 |
+| Concatenated from each `Observation.comment` where the Observation is referenced in <br/> `dianosticReport.result` and coded as a comment note                                             | `AGGREGATE_COMMENT_SET`                                 |
```

Comment:
Worth mentioning that comment note is `37331000000100`?

Reply:
Yes done